### DISCUSSION TITLE
FluentOverlay does not appear after closing a dialog

### DISCUSSION THREAD

**Discussion by NWierling** [microsoft/fluentui-blazor#1605] [Q&A] [answered]
Hello everyone :)

I have a dialog in which data can be entered that I would like to upload afterwards. If I set isLoading to true after closing the dialog, the FluentOverlay does not appear. The overlay works in the other methods for loading the data, but not in combination with the dialog. Can someone help me what I am doing wrong?

Specs -> InteractiveServerRenderMode(prerender: false)

Code:
![image](https://github.com/microsoft/fluentui-blazor/assets/136691532/ef5dddc9-4e05-4ad5-98d8-3dfdf6ca7223)

![image](https://github.com/microsoft/fluentui-blazor/assets/136691532/c1728afe-6677-4da3-a645-3971993f4201)

Greetings Nils

**Comment by vnbaaij** [comment, marked as answer]
Maybe you need a `StateHasChaged();` after setting `isLoading` to true?

**Reply by NWierling** [reply]
I thought I had tested it :D apparently not. Works - thanks for your help
